server/src/api/comment.rs

```diff
@@ -528,6 +540,14 @@ impl Perform for Oper<CreateCommentLike> {
         Ok(like) => like,
         Err(_e) => return Err(APIError::err("couldnt_like_comment").into()),
       };
+
+      if like_form.score == 1 {
+        comment.send_like(&user, &conn)?;
+      } else if like_form.score == -1 {
+        comment.send_dislike(&user, &conn)?;
+      }
```

Comment:
Probably cleaner to use an enum for upvote/downvote instead of this? Not really related to this PR though.